.ado/windows-vs-pr.yml

```diff
@@ -80,7 +80,8 @@ jobs:
           testAssemblyVer2: |
             Microsoft.ReactNative.Cxx.UnitTests/Microsoft.ReactNative.Cxx.UnitTests.exe
             Mso.UnitTests/Mso.UnitTests.exe
-            Microsoft.ReactNative.Managed.UnitTests/Microsoft.ReactNative.Managed.UnitTests/Microsoft.ReactNative.Managed.UnitTests.exe
+          # Managed unit tests crash with .net native https://github.com/Microsoft/vstest/issues/3838
+          # Microsoft.ReactNative.Managed.UnitTests/Microsoft.ReactNative.Managed.UnitTests/Microsoft.ReactNative.Managed.UnitTests.exe
```

Comment:
👀

Reply:
Does this disable in all circumstances?

Reply:
I know, I tried to enable them, but .net native still isn't happy. @vmoroz is working on getting us off the c# reflection that causes this.

Reply:
Can we at least specialize to release mode? We don't use .Net native for debug, right?

Reply:
I will enable it in another PR. I hadn't realized I'd missed it in the main VS2019 PR, and thought it would be easy to fit into this one. I'd rather not have it affect getting arm64 building. Now that I'm able to repro it locally and figured out that it's a crash, not whatever the agent logs are saying, I'm going to see if I can fix / specialize the actual tests

Reply:
#4515